gator_grouper/settings.py

```diff
@@ -48,6 +44,37 @@ def find_or_create_secret_key():
     return key
 
 
+def find_oauth_secret_key():
```

Comment:
I think you need to add some way to set this from an environment variable as well, so that it can be specified for Travis, as that is currently why it is failing.